HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2895,6 +2900,24 @@ def self.apply_hvac_sizing(hpxml_bldg, weather, cfa)
       htg_sys = hvac_system[:heating]
       clg_sys = hvac_system[:cooling]
 
+      # Fan W/cfm adjustment
+      airflow_max = get_airflow_max(htg_sys, clg_sys)
+      if !airflow_max.nil? # FIXME: we probably want a better flag here, right?
+        hpxml_bldg.heat_pumps.each do |heat_pump|
+          next unless [HPXML::HVACTypeHeatPumpAirToAir].include? heat_pump.heat_pump_type # FIXME: to what type(s), or when, should this apply?
+
+          v_baseline = airflow_max
+          v_upgrade = [hvac_sizing_values.Heat_Airflow.round, hvac_sizing_values.Cool_Airflow.round].max
+
+          p_int = v_baseline * heat_pump.fan_watts_per_cfm
+          p_upgrade = p_int * (v_baseline / v_upgrade)**3
+          fan_watts_per_cfm_adjusted = p_upgrade / v_upgrade
+
+          heat_pump.fan_watts_per_cfm = fan_watts_per_cfm_adjusted
+          heat_pump.fan_watts_per_cfm_isdefaulted = true
+        end
+      end
+
```

Comment:
Should this happen in `hvac_sizing.rb`? Would need to make sure that it happens _before_ `Heat_Airflow` and `Cool_Airflow` are potentially adjusted?

Reply:
Moved to `hvac_sizing.rb`.